src/main.cpp

```diff
@@ -254,13 +266,13 @@ main(int argc, char *argv[])
     }
 #endif
 
-    if (parser.isSet(debugOption))
-        nhlog::enable_debug_log_from_commandline = true;
-
     try {
-        nhlog::init(QStringLiteral("%1/nheko.log")
-                      .arg(QStandardPaths::writableLocation(QStandardPaths::CacheLocation))
-                      .toStdString());
+        std::string level =
+          (parser.isSet(debugOption) ? "trace," : "") + parser.value(logLevel).toStdString();
```

Comment:
I think the loglevel option should overwrite the debug option, i.e.:

```suggestion
        std::string level =
          parser.isSet(loglevel) ? parser.value(logLevel).toStdString() : (parser.isSet(debugOption) ? "trace" : ""));
```

Reply:
Levels set by --loglevel already override the --debug option. For example, `--debug --loglevel db=error` enables verbose logging for everything except db.

Or do you want --debug to be ignored in cases where someone uses both?

Reply:
We might also consider deprecating or removing the --debug option, since --loglevel offers a superset of its functionality. I kept it for backward compatibility, since I don't know who is using it.

Reply:
I think it should completely override the debug option. Otherwise it would be too confusing for me. I would not remove the debug option for now, since it is a bit of a pain to figure out what version people use before asking them for logs.

Reply:
Okay, then. I'll do that.